package.json

```diff
@@ -107,8 +107,5 @@
     "stylelint-order": "^6.0.1",
     "vinyl": "^3.0.0",
     "wait-on": "^7.0.1"
-  },
-  "overrides": {
-    "jest-environment-node": "^29.4.0"
```

Comment:
Reverted this temporary fix as it's resolved in `jest@29.4.1` instead

* https://github.com/smooth-code/jest-puppeteer/issues/512